docs/learning-powershell/README.md

```diff
@@ -31,7 +31,7 @@ At the end of this exercise, you should be able to launch the PowerShell session
 
 [inst-linux]: ../installation/linux.md
 [inst-win]: ../installation/windows.md
-[inst-macos]: ../installation/macos.md
+[inst-macos]: https://github.com/PowerShell/PowerShell-Docs/blob/staging/reference/docs-conceptual/setup/Installing-PowerShell-Core-on-macOS.md
```

Comment:
Please use `https://docs.microsoft.com/en-us/powershell/scripting/setup/installing-powershell-core-on-macos?view=powershell-6` instead.

Reply:
Please use `https://docs.microsoft.com/powershell/scripting/setup/installing-powershell-core-on-macos?view=powershell-6` (i.e. without the `en-us` culture) instead.